samples/notification-hubs-test-app/app/build.gradle

```diff
@@ -30,7 +30,7 @@ android {
 
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
-    implementation 'com.android.support:appcompat-v7:28.0.0'
+    implementation 'com.android.support:appcompat-v7:26.1.0'
```

Comment:
Do we want to target 26 or 28?  I'm assuming this puts it all back on 26

Reply:
targetSdkVersion is already 26 in the gradle file.

```
targetSdkVersion 26
```

I was getting build errors because the dependencies weren't inline with the SDK version.